src/main/java/org/ow2/proactive/catalog/service/WorkflowXmlManipulator.java

```diff
@@ -82,6 +82,28 @@
         }
     }
 
+    public byte[] replaceOrAddProjectNameOnJobLevel(final byte[] xmlWorkflow, String projectName) {
```

Comment:
Check that projectName is an empty space, if it is empty, do nothing

Reply:
The projectName value (empty string or not) provided by the user will be synchronized in the underlying xml file.

Reply:
if the parameter projectName is empty, don't we expect to actually remove the attribute (this is the behavior in the studio) ?

Reply:
Okey. I will update the PR